options/options.go

```diff
@@ -265,7 +281,7 @@ func MergeIAMRoleOptions(target IAMRoleOptions, source IAMRoleOptions) IAMRoleOp
 // Create a new TerragruntOptions object with reasonable defaults for real usage
 func NewTerragruntOptions() *TerragruntOptions {
 	return &TerragruntOptions{
-		TerraformPath:                  TerraformDefaultPath,
+		TerraformPath:                  TofuPath,
```

Comment:
We should probably stick with `terraform` as the default and `tofu` as the fallback. Otherwise, it's a backward incompatible change.